front-end/src/page/Form/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+// Local imports
+import './style.css'
+import Image from '../../static/image/image-info.png'
+
+function Form() {
+  return (
+    <div className="main">
+
+      {/* infomações */}
+      <div className="info">
+        <div className="text">
+          <div>
```

Comment:
I think that this div around the text isn't needed! You can apply the style in the own text tag.

```
<h1 className='my-class-here'>A simple text</h1>
```